test/connect_params.json

```diff
@@ -0,0 +1,105 @@
+[{
```

Comment:
include this file in test/kafka_connect_data_enrichment_tests folder as it specific to that test

Reply:
this file should be deleted.... not using it anymore :)